cli/tests/common/test_environment.rs

```diff
@@ -262,6 +266,16 @@ impl TestEnvironment {
         self.config_path = config_path.into();
     }
 
+    #[allow(dead_code)]
+    pub fn set_user_name(&mut self, user_name: String) {
+        self.user_name = user_name;
+    }
+
+    #[allow(dead_code)]
+    pub fn set_user_email(&mut self, user_email: String) {
+        self.user_email = user_email;
+    }
```

Comment:
You can override `JJ_USER`/`JJ_EMAIL` instead of configuring `TestEnvironment`.

```rust
test_env.run_jj_with(|cmd| cmd.current_dir(repo_root).env(..))
``

Reply:
Neat, thanks for pointing that out!